src/components/WalletAssetsHeadline.vue

```diff
@@ -0,0 +1,234 @@
+<template>
```

Comment:
please cover this component using jest

Reply:
wrote the simplest one with snapshot and props checking, more exhaustive test requires more time to fix arising issues during mount in all subsequent components